pages/login.vue

```diff
@@ -103,28 +107,6 @@ export default Vue.extend({
     resetCurrentUser(): void {
       this.currentUser = {};
       this.loginAsOther = false;
-    },
-    async getLoginDisclaimer(): Promise<void> {
-      try {
-        const brandingData = (await this.$api.branding.getBrandingOptions())
-          .data;
-        this.disclaimer = brandingData.LoginDisclaimer || '';
-      } catch (error) {
-        this.pushSnackbarMessage({
-          message: this.$t('unableGetServerConfiguration'),
-          color: 'error'
-        });
-      }
-    },
-    async getUsers(): Promise<void> {
-      try {
-        this.publicUsers = (await this.$api.user.getPublicUsers({})).data;
-      } catch (error) {
-        this.pushSnackbarMessage({
-          message: this.$t('unableGetPublicUsers'),
-          color: 'error'
-        });
-      }
```

Comment:
Why are we removing all this error catching?

Reply:
`fetch()` does it for us.

Reply:
But it doesn't display an error message. Imo we shouldn't silence errors but make them easier to understand for users.

Reply:
I guess what we need is a watcher for $fetchState.error that pushes the snackbar

Reply:
I was thinking of finally making that error modal I've been talking about, and generally using more visual indicators for errors than a simple snackbar (For example, replace the itemgrid on the library page with the error message), but that all felt like further PRs to improve things, rather than trying to cram it all in this one and essentially touching all the files and making a 1k lines PR.

Reply:
@MrTimscampi Okay, but don't forget about it 👍🏻